src/hotspot/share/interpreter/invocationCounter.hpp

```diff
@@ -68,7 +68,7 @@ class InvocationCounter {
   // Accessors
   bool carry() const               { return (_counter & carry_mask) != 0; }
   uint count() const               { return _counter >> number_of_noncount_bits; }
-  uint limit() const               { return CompileThreshold; }
+  uint limit() const               { return (uint)CompileThreshold; }
```

Comment:
CompileThreshold is an intx, and can be set by users on the command line. Casting it to uint looks wrong. This method seems to be used in only one place (invocationCounter.cpp:69), so changing the return type of the method to intx and fixing the print statement where it is used could be a better fix.

Reply:
That was a good suggestion.  Thank you!